plopfile.js

```diff
@@ -61,7 +68,9 @@ module.exports = function(plop) {
     description: 'Create a functional react component',
     prompts: componentPrompts,
     actions: function(data) {
-      const cwd = process.cwd();
+      const [path, componentName] = extractPathAndComponentName(data.name);
+      data.name = componentName;
+      const cwd = `${process.cwd()}/${path}`;
```

Comment:
This needs to be done in two places because component is not the same as class component.

Reply:
Done!